solitude/tests/live.py

```diff
@@ -0,0 +1,73 @@
+import os
+
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+from django.test import LiveServerTestCase
+from django.test.utils import override_settings
+
+from nose.plugins.attrib import attr
+
+from curling import lib
+
+from solitude.base import getLogger
+
+log = getLogger('s.tests')
+
+# Tests run using soltiude/settings/test.py. But because we are talking to the
+# live Braintree instance, we need to get the Braintree config from somewhere
+# without adding it to the source.
+#
+# For developers working locally, it's likely to be in solitude.settings.local.
+#
+# If it's not there, let's fall back to the environment variables, suitable for
+# Travis CI.
+#
+# If not found, raise an error.
+try:
+    from solitude.settings import local
+except ImportError:
+    local = None
+
+# Values that we are going to look for in local, then env.
+envs = {
+    'BRAINTREE_MERCHANT_ID': '',
+    'BRAINTREE_PUBLIC_KEY': '',
+    'BRAINTREE_PRIVATE_KEY': ''
+}
+for key in envs.keys():
+    value = ''
+    if local:
+        log.info('Found {0} from local.py.'.format(key))
+        value = getattr(local, key)
+
+    if not value:
+        log.info('Found {0} from environment.'.format(key))
+        value = os.environ.get(key)
```

Comment:
Why is this necessary? The [base settings](https://github.com/andymckay/solitude/blob/378/solitude/settings/base.py) file already gets values from the environment like ``BRAINTREE_MERCHANT_ID = os.getenv('BRAINTREE_MERCHANT_ID', '')``